bindings/core/src/method/account.rs

```diff
@@ -50,7 +50,7 @@ pub enum AccountMethod {
     DeregisterParticipationEvent { event_id: ParticipationEventId },
     /// Generate new unused addresses.
     /// Expected response: [`GeneratedAddress`](crate::Response::GeneratedAddress)
```

Comment:
The merge commit changed this back
```suggestion
    /// Expected response: [`GeneratedEd25519Addresses`](crate::Response::GeneratedEd25519Addresses)
```